flows/inspector: add button to open flow inspector (#12656)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/flow/inspector.md

Be aware that when running a flow with the inspector enabled, the flow is still executed normally. This means that for example, a [User write](../stages/user_write.md) stage _will_ write user data.
:::

> The inspector is accessible to users that have been granted the [permission](../../../users-sources/access-control/permissions.md) **Can inspect a Flow's execution**, either directly or through a role. Superuser can always inspect flow executions.

Reviewer: ```suggestion
The inspector is accessible to users that have been granted the [permission](../../../users-sources/access-control/permissions.md) **Can inspect a Flow's execution**, either directly or through a role. Superusers can always inspect flow executions.
```

Reviewer: And we still want to add more, right, about how they also need View permissions on that specific flow?

Author: I think you're mixing that up with app entitlements, in this case they dont need any other permission

When developing authentik with the debug mode enabled, the inspector is enabled by default and can be accessed by both unauthenticated users and standard users. However the debug mode should only be used for the development of authentik. So unless you are a developer and need the more verbose error information, the best practice for using the flow inspector is to assign the permission, not use debug mode.

> Starting with authentik 2025.2, when the user can access the inspector, a button is shown in the top right of the [default flow executor](./executors/if-flow.md) which opens the flow inspector.

Reviewer: ```suggestion
Starting with authentik 2025.2, for users with appropriate permissions to access the inspector a button is shown in the top right of the [default flow executor](./executors/if-flow.md) which opens the flow inspector.
```

:::info
Troubleshooting:

> - If the flow inspector does not launch and a "Bad request" error displays, this is likely either because you selected a flow that has a policy bound directly to it that prevents access, so the inspector won't open because the flow can't be executed.

Reviewer: You say "either" but then only give one possible reason. Is the other reason that the user does not have view permissions on the flow?

If yes, then maybe this wording:

- If the flow inspector does not launch and a "Bad request" error displays, this is likely either because you selected a flow that has a policy bound directly to it that prevents access (so the inspector won't open because the flow can't be executed) or because you do not have view permission on that specific flow.
